oic.r.doxm.raml

```diff
@@ -75,12 +74,10 @@ traits:
       Creates the DOXM resource data
     body:
       application/json:
-        schema: Doxm
+        schema: Doxm-Update
```

Comment:
This is a comment for lines 65 and 66.  I'm not sure that a 201 Created response is valid.  Can we remove these two lines??

Reply:
This applies to the entire PUT section, which can't be used to CREATE anything since /doxm doesn't support CREATE.  I wonder if we should just remove lines 72 through 93.  Maybe ask Mark for clarification as I've never seen PUT specified in any other RAML files...